test/specs/controller.bar.tests.js

```diff
@@ -823,10 +887,10 @@ describe('Bar controller tests', function() {
 		var meta0 = chart.getDatasetMeta(0);
 
 		[
-			{b: 290, w: 83 / 2, x: 63, y: 161},
```

Comment:
Similar to a talk we had, can we keep it in this form: `w: 92 / 2` (sample size / number of stacks)